packages/sp1-ics07-tendermint-utils/src/rpc.rs

```diff
@@ -85,7 +85,8 @@ impl TendermintRpcExt for HttpClient {
             signed_header.header().proposer_address,
         )?;
 
-        let next_validator_response = self.validators(height + 1, Paging::All).await?;
+        let next_height = height.increment();
```

Comment:
I don't like these variables which are not used anywhere else, nor having to go into increment to understand the code

Reply:
fixed